config/config_test.go

```diff
@@ -14,6 +14,8 @@ import (
 func TestDefaults(t *testing.T) {
 	v := viper.New()
 	SetupViper(v, "")
+	v.SetConfigType("yaml")
+	v.ReadConfig(bytes.NewBuffer(requiredConfig))
 	cfg, err := New(v)
```

Comment:
Please rollback the modifications made to this file. If you add the required  entries to `config/config.go` discussed in our comment above, these changes are unnecessary

Reply:
Hi, we do not have default endpoints. All our endpoints are unique per publisher, the bidding and the user sync endpoint. I had to change this in order to pass because of our context. Do you mean that is required that we have to set some default endpoints?
If yes:
- we can set dummy urls that does nothing
- we can set a base url like https://verizonmedia.com and in the config processing to append some publisher specific URL paths. 
Which one will be accepted?

Reply:
Good day. If there's no default endpoint we can do 
```
v.SetDefault("adapters.foo.disabled", true)
```
Instead of 
```
v.SetDefault("adapters.foo.endpoint", URL2)
```
Inside the `func SetupViper(v *viper.Viper, filename string)` function of `config/config.go`. Please let me know if this works.

Reply:
The issue here is if the URL is dynamic, based on the content of the openrtb request, configure a base URL and modify it in your adapter to be the final URL. If the URL depends on the PBS host (the PBS hosting entity should enter something unique in the config to get your adapter working properly), then the adapter should default to disabled. The idea is that PBS should come up in a workable state even if not supplied with any configuration.

Reply:
I see, the URL is something the publisher will have to setup in his config. I'll flag it disabled as suggested because  without this new adapter config properties the server does not start. Just as a notification I would like to say that we got through rebranding and we'll change the name of the adapter from 'onemobile' to 'verizonmedia'.

Reply:
Just one note on terminology to help avoid confusion. "publisher" would be the actual source of ad traffic, using prebid to increase revenue. "PBS host" would be the entity running the instance of prebid server. They can both be the same thing, but won't always be the same.